### PR TITLE
update README.md about sample of sklearn for testing the SKLearn Serv…

### PR DESCRIPTION
**Pull request by jiaqianjing**
When I followed the userguide about testing sklearnserver locally , it failed. 
`--model_dir`  should  be set dir not file of model, like model.joblib

### CODE REVIEW

**Comment by jiaqianjing** [comment]
> @googlebot I fixed it.

@googlebot I fixed it.

File: docs/samples/sklearn/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ dump(clf, 'model.joblib')
 Then, we can install and run the [SKLearn Server](../../../python/sklearnserver) using the generated model and test for prediction. Models can be on local filesystem, S3 compatible object storage, Azure Blob Storage, or Google Cloud Storage.
 
 ```shell
-python -m sklearnserver --model_dir model.joblib --model_name svm
+python -m sklearnserver --model_dir ./  --model_name svm
```
Comment (yuzisun) [inline docs/samples/sklearn/README.md:19]: @jiaqianjing good catch! Can you help add a comment that this should be pointing a model directory which contains the actual model?

Comment (jiaqianjing) [reply docs/samples/sklearn/README.md:19]: Done!

**Comment by yuzisun** [comment]
Thanks @jiaqianjing for the fixing!

/lgtm
/approve

**Pull request merged**